api/net/Areas/Admin/Controllers/MediaAnalyticsController.cs

```diff
@@ -0,0 +1,134 @@
+using System.Net;
+using System.Net.Mime;
+using System.Text.Json;
+using Microsoft.AspNetCore.Http.Extensions;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Swashbuckle.AspNetCore.Annotations;
+using TNO.API.Areas.Admin.Models.MediaAnalytics; 
+using TNO.API.Models;
+using TNO.Core.Exceptions;
+using TNO.DAL.Services;
+using TNO.Entities;
+using TNO.Entities.Models;
+using TNO.Keycloak;
+
+namespace TNO.API.Areas.Admin.Controllers;
+
+/// <summary>
+/// MediaTypeController class, provides MediaType endpoints for the api.
+/// </summary>
+///[ClientRoleAuthorize(ClientRole.Administrator)]
+[ApiController]
+[Area("admin")]
+[ApiVersion("1.0")]
+[Route("api/v{version:apiVersion}/[area]/mediaanalytics")]
```

Comment:
I'm not a fan of multiple words pasted together.  I'd change to `media-analytics`.  This isn't critical though.